client/src/App.js

```diff
@@ -243,7 +254,7 @@ class App extends Component {
             path="/home"
             exact
             render={routerProps => (
-              <HomePage handle_logout={this.handleLogout} {...routerProps} {...this.state} />
+              <HomePage loggedIn={this.state.logged_in} handle_logout={this.handleLogout} {...routerProps} {...this.state} />
```

Comment:
I think you can delete this whole route, since the new homepage route is just `/`

Reply:
In my comment below where I say "in the route", I mean in this function.

Reply:
Oh and rather than render a different component, can you redirect the user to a different page?